client/client.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/nspcc-dev/neofs-api-go/v2/rpc/client"
 	neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
 	neofsecdsa "github.com/nspcc-dev/neofs-sdk-go/crypto/ecdsa"
+	"github.com/nspcc-dev/neofs-sdk-go/internal/util"
```

Comment:
No. Never. Ever. Have `libmisc` or `libutil`.

Reply:
No-no-no, _anything else_ but util/misc. `addrutil` is already 15× better. Some other names can be 50× better.

Reply:
Any `libmisc`/`libutil` quickly becomes a code trash bin. We don't want that.

Reply:
got it now, and agree. How bout `netutil`? like https://pkg.go.dev/golang.org/x/net/netutil

Reply:
`uriutil.Parse` maybe. `net` or `uri` or `addr` are too generic for the thing.

Reply:
`uriutil` now